qubes/tools/qvm_start.py

```diff
@@ -133,13 +133,17 @@ def main(args=None):
     if args.debug:
         vm.debug = args.debug
 
-    try:
+    if args.debug:
         vm.start(
             preparing_dvm=args.preparing_dvm,
             start_guid=args.start_guid)
-    except qubes.exc.QubesException as e:
-        parser.error_runtime('Qubes error: {!r}'.format(e))
-
+    else:
+        try:
+            vm.start(
+                preparing_dvm=args.preparing_dvm,
+                start_guid=args.start_guid)
+        except qubes.exc.QubesException as e:
+            parser.error_runtime('Qubes error: {!r}'.format(e))
```

Comment:
@marmarek No it shouldn't. Because it will end up there only if there is no exception thrown during any vm.start() calls.